modules/p2/ding_sharer/ding_sharer.module

```diff
@@ -95,21 +96,28 @@ function template_preprocess_ding_sharer(&$variables) {
  */
 function theme_ding_sharer($variables) {
   $out = '';
+  $anchor_class = '';
 
-  if (!empty($variables['label'])) {
+  if (!empty($variables['label']) && $variables['type'] == 'normal') {
```

Comment:
This has normal (all lowercase). The default value is Normal (first letter uppercase). That does not seem right?

Reply:
Are you sure that you that this works right? I would expect that ` $variables['type']` refers to the value of the option ("normal") - not the textual version ("Normal"). The only place I would expect to see "Normal" would be in the textual version. Now it is spreading.

Reply:
Youre are right! Thanx for clarifiyng!